source/PlayerInfo.h

```diff
@@ -232,6 +237,13 @@ class PlayerInfo {
 	const ConditionsStore &Conditions() const;
 	std::map<std::string, std::string> GetSubstitutions() const;
 
+	// Get and set the "tribute" that the player receives from dominated planets.
+	bool SetTribute(const Planet * planet, int64_t payment);
+	bool SetTribute(const std::string &planetTrueName, int64_t payment);
+	const std::map<const Planet *, int64_t> GetTribute() const;
+	int64_t GetTributeTotal() const;
+	int64_t GetTributeForPlanet(const std::string &planetTrueName) const;
```

Comment:
still not sure about these 3 methods being public

Reply:
I already removed `GetTributeForPlanet` from the cpp file, and I will remove it from this .h file as well. GetTribute and GetTributeTotal are used in BankPanel, Planet and in PlayerInfoPanel, so those I would like to keep as public methods.